samples/language/Ternary.qs

```diff
@@ -0,0 +1,15 @@
+/// # Sample
+/// Ternary
+///
+/// # Description
+/// Q# supports conditional expressions (`if` expressions) without the use of a ternary operator.
+namespace MyQuantumApp {
+    @EntryPoint()
+    operation Main() : Unit {
+        let fahrenheit = 40;
+
+        // `if` can also be used as an expression, to conditionally return a value.
+        // This emulates the behavior of the ternary operator in languages like Python.
+        let absoluteValue = if fahrenheit > 0 { fahrenheit } else { fahrenheit * -1 };        
```

Comment:
```suggestion
        // Q#'s ternary operator is of the form: `cond ? ifTrue | ifFalse`.
        // If the `cond` expression evaluates to true, then the conditional 
        // (ternary) expression evaluates to the `ifTrue` expression; otherwise,
        // it evaluates to the `ifFalse` expression.
        // For example:
        let absoluteValue = fahrenheit > 0 ? fahrenheit | fahrenheit * -1;
```